build.gradle

```diff
@@ -2673,13 +2673,6 @@ project(':streams') {
             ':streams:integration-tests',
             ':streams:test-utils:test',
             ':streams:streams-scala:test',
-            ':streams:upgrade-system-tests-0100:test',
-            ':streams:upgrade-system-tests-0101:test',
-            ':streams:upgrade-system-tests-0102:test',
-            ':streams:upgrade-system-tests-0110:test',
-            ':streams:upgrade-system-tests-10:test',
-            ':streams:upgrade-system-tests-11:test',
-            ':streams:upgrade-system-tests-20:test',
```

Comment:
Probably, this part can also be removed.

https://github.com/apache/kafka/blob/50c15b94c94fbe8f964703c057963b38100b0bd6/streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java#L282-L322